### ISSUE TITLE
Inference mode CPU slow 

### ISSUE THREAD

**Issue by Houd1ny** [matterport/Mask_RCNN#866]
Guys thank you for this project, this repo has a very good API!!
I followed "Splash of Color: Instance Segmentation with Mask R-CNN " tutorial and implemented for my own dataset.
My trained model runs in 13 seconds on CPU(i7 8 cores) for one image.
I would like it to work 3-4 seconds.
I have some ideas how to make this work:
1.Compile tensorflow for current process architecture.
2.Use different backbone resnet50 or mobilenet2 (https://github.com/GustavZ/Mobile_Mask_RCNN/)
3.Use smaller images size(How to do this?  will pretraining work with different sizes?)
4.NN pruning
Is there other thinks I could do?
Is is achievable tasks?
Thanks in advance.

**Comment by fastlater** [comment]
@Houd1ny  did you achieve any good result with those changes?

**Comment by Houd1ny** [comment]
@fastlater yes, mobile net works faster

**Comment by cooleel** [comment]
@Houd1ny  Could you please share the config file of running the inference mode on a CPU?
I met some problem using my trained model to predict on a CPU, but the same script runs well on GPU. So I wonder if you have some changes in the config. Thanks!

**Comment by ectg** [comment]
Hi @Houd1ny, Did you get this to work? 

I need inference to run at 60 fps on CPU, and it seems unlikely that MRCNN can provide that type of performance.

**Comment by Houd1ny** [comment]
@ectg Hi
No, for my task I just switched to GPU. 
For your case, I don`t know if it will be possible to run it in real time or it will cost a lot.
You should consider other models.
Search for something like this
https://www.learnopencv.com/deep-learning-based-object-detection-using-yolov3-with-opencv-python-c/

**Comment by pierricC** [comment]
Hello @cooleel, I face exactly the same issue, did you manage to make it work ?

**Comment by pierricC** [comment]
Hello @cooleel, I face exactly the same issue, did you manage to make it work ?

**Comment by pierricC** [comment]
Hello @cooleel, I face exactly the same issue, did you manage to make it work ?

**Comment by pierricC** [comment]
Hello @cooleel, I face exactly the same issue, did you manage to make it work ?

**Comment by Adithia99** [comment]
guys do have the codes , how to detect videos with FPS? i can detect it in real time but dont know formula how to calculate FPS
